File: pgm/ABC_Java/ABC_175_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] S = reader.readLine().split ("");
        final String RAINY = "R";
        final int one = 1;
        final int zero = 0;
        int num, ans;
        num = zero;
        ans = zero;
        for (int i = zero; i < S.length; i++) {
            if (RAINY.equals (S[i])) {
                num += one;
            } else {
                ans = Math.max (ans, num);
                num = zero;
            }
        }
        System.out.println (Math.max (ans, num));
        return;
    }
}

File: pgm/ABC_R/ABC_175_A.r
con <- file(description = "stdin", open = "r")
arr <- readLines(con = con)
close(con)
RAINY <- "R"
one <- 1
zero <- 0
blank <- ""
S <- strsplit(arr[one], split = blank)[[one]]
num <- zero
ans <- zero
for (c in S) {
    if (c == RAINY) {
        num = num + one
    } else {
        ans <- max(ans, num)
        num <- zero
    }
}
max(ans, num) |>
    cat("\n", sep = blank)
q("no")